src/ShadowsOfShadows.csproj

```diff
@@ -115,21 +124,12 @@
     <Compile Include="Items\WizardsCape.cs" />
     <Compile Include="Items\WizardsHood.cs" />
     <Compile Include="Items\WoodenShield.cs" />
+    <Compile Include="Serialization\GameState.cs" />
     <Compile Include="TestData\TestRooms.cs" />
   </ItemGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
+  <ItemGroup />
   <ItemGroup>
-    <Folder Include="Entities\" />
```

Comment:
Jeśli nie ma potrzeby to bym tego nie usuwał. Wiem, że prawdopodobnie IDE ci to zrobiło automatycznie, ale lepiej to cofnąć.
Zapoznaj się z poleceniem `git rebase -i` oraz `git add -p` ;)